README.md

```diff
@@ -1,100 +1,125 @@
+# Introduction
 ## What is Baskerville?
 
 Manual identification and mitigation of (DDoS) attacks on websites is a difficult and time-consuming task with many challenges. This is why Baskerville was created, to identify the attacks directed to Deflect protected 
 websites as they happen and give the infrastructure the time to respond properly. Baskerville is an analytics engine that leverages Machine Learning to distinguish between normal and abnormal web traffic behavior. 
+
 In short, Baskerville is a Layer 7(application layer) DDoS attack mitigation tool.
 
-## What is Baskerville client
+## What is Baskerville client?
+
+Baskerville client is a client module which:
+* Processes nginx web server logs and calculates statistical features.
+* Sends features to a clearing house instance of Baskerville.
+* Receives predictions from a clearing house for every IP.
+* Issues challenge commands for every malicious IP in a separate Kafka topic.
+* Monitors attacks in Grafana dashboards.
+
+# System requirements
+* Linux server with Docker already installed.
+* A minimum of 8GB of RAM is suggested.
+* Allow port number `29092` on the host's firewall for Kafka connections to and from the clearing house.
+
```

Comment:
* 29092 is only for connection **from** the clearinghouse to the client
* We need to allow port 3000 for Grafana as well.

Reply:
Thanks, that has been updated in 43419241fe11f2b16bf8467844a9ea14cc4ad7db.